spec/unit/defaults_spec.rb

```diff
@@ -188,8 +188,33 @@
     end
 
     it "raises an exception if facter-ng could not be loaded" do
-      allow(Puppet).to receive(:require).with('facter-ng').and_return(false)
+      allow(Kernel).to receive(:require).and_call_original
```

Comment:
I don't think this is needed as `Kernel` is not stubbed in other places.

Reply:
You're right, I'll update.